gson/src/main/java/com/google/gson/GsonBuilder.java

```diff
@@ -325,6 +327,30 @@ public GsonBuilder setFieldNamingStrategy(FieldNamingStrategy fieldNamingStrateg
     return this;
   }
 
+  /**
+   * Configures Gson to apply a specific number strategy during deserialization of {@link Object}.
+   *
+   * @param objectToNumberStrategy the actual object-to-number strategy
+   * @return a reference to this {@code GsonBuilder} object to fulfill the "Builder" pattern
+   * @see ToNumberPolicy#DOUBLE The default object to number strategy
```

Comment:
```suggestion
   * @see ToNumberPolicy#DOUBLE The default object-to-number strategy
```
To be consistent with the parameter description
Or change the parameter description to not use hyphens